### PR TITLE
Remove the Incubation description

### PR DESCRIPTION
**Pull request by SLSJL**
1. Remove the incubation description from the footer
2. Add the Apache logo on the footer area
3. Remove the incubation-related copyrights from the page bottom

### CODE REVIEW

**Review by Pil0tXia — APPROVED** [review]
LGTM, Good Job! Please add `Fixes #162` in the text~

**Review by xiaoyang-sde — APPROVED** [review]

File: src/theme/Footer/index.tsx [hunk 1]
```diff
@@ -78,37 +77,41 @@ function Footer(): JSX.Element | null {
...
-                        }}
-                      />
-                    ) : (
-                      <li key={item.href || item.to} className="footer__item">
-                        <FooterLink {...item} />
-                      </li>
-                    )))}
-                  </ul>
+            <div  className="col footer__col">
+              <img src='https://doris.apache.org/images/asf_logo_apache.svg' />
```
[the remark this answers is not in the document]
Comment (SLSJL) [reply src/theme/Footer/index.tsx:81]: Good spot, thanks.

**Comment by SLSJL** [comment]
> LGTM, Good Job! Please add `Fixes #162` in the text~

Ok, will close this request and submit a new one soon

**Comment by SLSJL** [comment]
Close to amend the PR title and the source of the Apache logo

**Pull request closed without merging**